openfisca_france/model/prestations/aide_mobilite.py

```diff
@@ -0,0 +1,377 @@
+        duree_de_contrat_3_mois_minimum = individu('contrat_de_travail_duree', period) >= parametres_amob.duree_de_contrat_minimum
+
+        reprises_cdd_ctt_eligibles = reprises_types_activites_cdd_ctt * duree_de_contrat_3_mois_minimum
+
+        types_et_duree_activite_eligibles = (((reprises_types_activites_cdi + reprises_cdd_ctt_eligibles) * (en_reprise_emploi + en_entretien_embauche))
+                                            + (reprises_types_activites_formation
+                                               * en_formation
+                                               * formation_validee
+                                               * formation_financee
+                                               * periode_formation_eligible))
+
+        activites_eligibles = (types_et_duree_activite_eligibles + activites_en_recherche_emploi_eligibles)
+
+        #  3
+        categories_eligibles = individu('aide_mobilite_categories_demandeur_emploi_eligibles', period)
+
+        #  4
+        montants_allocations_eligibles = individu('aide_mobilite_allocations_eligibles', period)
+
+        #  5
+        lieux_activite_eligibles = not_(individu('lieu_emploi_ou_formation', period) == TypesLieuEmploiFormation.non_renseigne)
+
+        #  6
+        lieu_de_residence = individu.menage('residence', period)
+        temps_de_trajet = individu('aide_mobilite_duree_trajet', period)
+        distance_aller_retour = individu('distance_aller_retour_activite_domicile', period)
+
+        temps_de_trajet_min = parametres_amob.duree_trajet_minimum
+        reside_en_metropole = lieu_de_residence == TypesLieuResidence.metropole
+        residence_renseignee = not_(lieu_de_residence == TypesLieuResidence.non_renseigne)
+
+        distances_et_durees_aller_retour_eligibles = (((distance_aller_retour > parametres_amob.distance_minimum.metropole) * reside_en_metropole)
+                                                    + ((distance_aller_retour > parametres_amob.distance_minimum.hors_metropole) * (not_(reside_en_metropole) * residence_renseignee))
+                                                    + ((temps_de_trajet > temps_de_trajet_min) * residence_renseignee))
+
+        #  7
+        dispositifs_formations = individu('dispositifs_formation', period)
+        dispositifs_formations_eligibles = dispositifs_formations == DispositifsDeFormation.autre
+
+        eligibilite_amob = (contextes_eligibles
+                       * activites_eligibles
+                       * categories_eligibles
+                       * montants_allocations_eligibles
+                       * lieux_activite_eligibles
+                       * distances_et_durees_aller_retour_eligibles
+                       * dispositifs_formations_eligibles)
+
+        return eligibilite_amob
+
+
+class aide_mobilite(Variable):
+    value_type = float
+    entity = Individu
+    label = "Calcul du montant de l'aide à la mobilité - AMOB"
```

Comment:
Tout n'est-il pas calcul par ici ? 😁 
```suggestion
    label = "Montant de l'aide à la mobilité de Pôle emploi - AMOB"
```